.vib/elasticsearch/goss/goss.yaml

```diff
@@ -0,0 +1,28 @@
+process:
```

Comment:
Although we usually use Cypress to tests the asset's API, AFAIK the `GET /` request returns the ES version. I think we should add that test here and see it as a way to check the correct asset version more than a functionality test. WDYT?

Reply:
```
➜  charts git:(add-elasticsearch-tests) ✗  k exec -it elasticsearch-master-0 -- bash
Defaulted container "elasticsearch" out of: elasticsearch, sysctl (init)
I have no name!@elasticsearch-master-0:/$ GET /
bash: GET: command not found
```
Good idea, but I'm not sure if I'm getting the syntax right. I'll check the docs. Are you 100% if this is the command?

Reply:
I could be missing something for sure. I know we use a similar test on BTS, but I didn't look at it in detail sorry. It's under the verification tests if you want to check.

Reply:
Thanks for the idea! I've added this test instead of the `java` test.